Wordle.Api/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers;
+[Route("api/[controller]")]
+[ApiController]
+public class PlayerController : ControllerBase
+{
+    public class PlayerPost
+    {
+        public string Name { get; set; } = "Anonymous";
+        public int Guesses { get; set; }
+        public int Seconds { get; set; }
+    }
+
+    private readonly PlayerService _service;
+    public PlayerController(PlayerService service)
+    {
+        _service = service;
+    }
+
+    [HttpGet]
+    public IEnumerable<Player> Get()
+    {
+        Player[] players = _service.GetPlayers().ToArray();
+        if(players.Length < 10)
+            return players.ToList().GetRange(0, players.Length);
+        else
+            return players.ToList().GetRange(0, 10);
+    }
```

Comment:
For the get method, dont forget to order the results in some way. the way we did it was using `(7 - player.AverageAttempts) * player.GameCount)`